src/relentless/simulate/md.py

```diff
@@ -118,9 +118,10 @@ class RunLangevinDynamics(_Integrator):
 
     """
 
-    def __init__(self, steps, timestep, T, friction, seed, analyzers=None):
+    def __init__(self, steps, timestep, T, barostat, friction, seed, analyzers=None):
```

Comment:
Can you add the `barostat=None` right before `analyzers` so it is an optional argument? Then, update and handle this accordingly in the HOOMD backend. That may fix some of your failing unit tests.